guides/v15-mdc-migration.md

```diff
@@ -0,0 +1,571 @@
+
+  TODO(mmalerba): replace this with a working image or embedded example:
+  ![alt_text](../images/slide-toggle-diff.png "Diagram showing overlapping slide-toggle touch targets")
+
+* The label is closer to the enabled toggle
+
+### Slider
+
+* Accessibility: sliders now work with mobile device screen readers.
+
+* To accommodate range sliders, the implementation has changed from the `<mat-slider>` element being
+  the form control to the `<mat-slider>` element containing 1-2 `<input>` elements (the slider
+  "thumbs") that act as the form control(s). The value, associated events (`input`, `change`), and
+  labels (`aria-label`) now live on the `<input>` elements instead.
+
+* Vertical sliders and inverted sliders are no longer supported, as they are not part of the
+  Material Design spec.
+
+* Range sliders are now supported. TODO(mmalerba): add more about this.
+
+### Snack Bar
+
+* For simple, text-based snack-bars, there are no significant changes.
+
+* For simple snack-bars with an action button, they use the MDC-based mat-button, so your app will
+  need to include the theme sass for the MDC-based button
+
+* For snack-bars that use custom structured content (i.e. calls to `MatSnackBar.openFromComponent`
+  and `MatSnackBar.openFromTemplate`), you  should use the following new directives to annotate your
+  content:
+  * `matSnackBarLabel` to mark the text displayed to users
+  * `matSnackBarActions` to mark the element containing the action buttons
+  * `matSnackBarAction` to mark individual action buttons
+  * If you do not specify any of these directives, it will treat the entire custom component /
+    template as text.
+
+* Tests that open a snack-bar now require calling `flush()` before attempting to access the content
+  of the snackbar.
+
+### Table
+
+* Every cell has `16px` left and right padding. Previously only a `24px` padding applied to the
+  leftmost and rightmost cells.
+
+* Header text size and color matches the data rows.
+
+* Text will not wrap by default; to enable text wrapping, apply the `white-space: normal` style to
+  the `<mat-cell>`.
+
+* Row height is `52px` instead of `48px`.
+
+* Cells have box-sizing `border-box` instead of `content-box`. This may affect  custom width styles.
+
+* The paginator property of the `MatTableDataSource` has a generic interface that matches most of
+  the paginator API. You may need to explicitly type the paginator to access the full API, e.g.
+  `new MatTableDataSource<MyData, MatPaginator>();`
+
+* MDC-based flex tables (`<mat-table>`) display borders on their cells instead  of the rows.
+
+* The last row of the table no longer has a bottom border since the table is assumed to be
+  surrounded by a border.
+
+* Flex table row height is set using `height` instead of `min-height`.
+
+### Tabs
+
+* Accessibility: `MatTabNav` now throws an error if the `[tabPanel]` input is not provided. The
+  `[tabPanel]` input and associated `mat-tab-nav-panel` component improve accessibility.
```

Comment:
This one is going to need some further elaboration (we changed the entire interaction pattern of nav tabs)

Reply:
Ok, I plan to have the people with the most context on each component go over this guide after and polish up their sections / make sure all relevant info is included. Adding a TODO in the mean time.